settings.gradle

```diff
@@ -1 +1,12 @@
-include ':app', ':cmplibrary', ':authexample', ':metaapp'
+include ":cmplibrary"
```

Comment:
to review
This is the common way for Gradle to declare projects that are in sub-folder